Week3/more_javascript.js

```diff
@@ -101,36 +101,22 @@ myVehicles.push("mopid");
 console.log("Amazing Joe's Garage, we service "+advertisement(myVehicles));
 
 /*12- Create an empty object.*/
-let teachers1 ={};
-let teachers2 = new Object();
-console.log(teachers1);
-console.log(teachers2);
+let firstMethod ={};
+let secondMethod = new Object();
+console.log(firstMethod);
+console.log(secondMethod);
 /* 13- Create an object that contains the teachers that you have had so far for the different modules.*/ 
-let teachers=[{
-   Name: 'Rob',
-},{
-   Name: 'Philipp',
-},{
-   Name: 'Unmesh',
-},{
-   Name: 'Yash',
-}]
+let teachers={
+   teacher: 'Rob, Philipp, Unmesh, Yash',
+ }
+
 console.log(teachers);
 /*14- Add a property to the object you just created that contains the languages that they have taught you.*/
-teachers=[{
-   Name: 'Rob',
-   language: 'HTML/CSS',
-},{
-   Name: 'Philipp',
-   language: 'HTML/CSS',
-},{
-   Name: 'Unmes',
-   language: 'CLI',
-},{
-   Name: 'Yash',
-   language: 'Javascript1',
-}]
-console.log(teachers);
+let teachers1={
```

Comment:
Ah, actually the old version of the code was correct! I misunderstood the assignment before, but after talking to Yash, I realized that the old way of your code is correct!

```
teachers=[{
   Name: 'Rob',
   language: 'HTML/CSS',
},{
   Name: 'Philipp',
   language: 'HTML/CSS',
},{
   Name: 'Unmes',
   language: 'CLI',
},{
   Name: 'Yash',
   language: 'Javascript1',
}]
console.log(teachers);
```